model/Licensing/Classes/SimpleLicensingText.md

```diff
@@ -0,0 +1,26 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SimpleLicensingText
+
+## Summary
+
+A license or addition that is not listed on the SPDX License List.
+
+## Description
+
+A SimpleLicensingText represents a License or Addition that is not listed on the SPDX License
+List at https://spdx.org/licenses, and is therefore defined by an SPDX data
+creator.
+
+## Metadata
+
+- name: SimpleLicensingText
+- SubclassOf: AnyLicenseInfo
```

Comment:
I'm not sure if we want `SimpleLicensingText` to be a subclass of `AnyLicenseInfo` - if someone is making a declared license or concluded license, should we restrict it to always be either a license expression string or an expanded license?  If you want to reference a custom license or addition, I think you would want to use a license expression string with the ID of the `SimpleLicensingText` and keep the `SimpleLicensingText` as a direct subclass of `Element`.  Otherwise, it would be more complex to parse in the simple case and it would be confusing when to use the `SimpleLicensingText` or the `CustomLicense`.

Reply:
I agree that `SimpleLicensingText` is better off subclassing `Element`.

Reply:
I also agree.

Reply:
done